message-markdown.md

```diff
@@ -0,0 +1,44 @@
+# Markdown in messages
+
+Deltachat is going to support the following subset of markdown:
+
+### `**bold**`
+
+### `__underline__`
+
+### `~~strikethrough~~`
+
+### `https://delta.chat` - Urls
+
+Make URLs clickable, but **don't** support the Link syntax `[Name](url)`
+because that could be used for misleading and as such be used in scams.
+
+### `` `inline-code` ``
+useful to send non markdown text in your message like source code snippets.
+Should get rendered in a monospace font and with a different background.
+ 
+### ` ``` fence code block ``` `
+
+```md
+Similar to `inline-code` but not inline and it may supports code highlighting.
+```
+` ```[lang?] [content]``` `
+ A bit modified from the common syntax to allow one liners.
+
+### `:emoji:`
+
+## Future:
+
+### `mailto:email@address.example.com`
```

Comment:
also here i suggest to use <mailto:email@address.example.com> instead, however, emails and most http-links are already now clickable on android without special formatting.

Reply:
The plan is to put every formating thing on this list that goes beyond plain-text. Maybe the naming of the file could be improved in that regard

Reply:
> also here i suggest to use [mailto:email@address.example.com](mailto:email@address.example.com) instead, however, emails and most http-links are already now clickable on android without special formatting.

plain email addresses are clickable on dc-android, but mailto: urls aren't :(